packages/ui/src/print-content/PrintContent.css

```diff
@@ -2,29 +2,26 @@
 .ui-printContent--printContainer {
   display: none;
   visibility: hidden;
+  overflow: hidden;
 }
 
-.ui-printContent--questionTable th,
-.ui-printContent--questionTable td {
+.ui-printContent--table th,
+.ui-printContent--table td {
   border: var(--print-content-table-th-td-border);
-  padding: var(--layout-padding-xsmall);
+  padding: var(--layout-padding-10);
+  font: var(--typography-regular-small-body);
   text-align: left;
   word-wrap: break-word;
-  font: var(--typography-regular-small-body);
+  break-inside: avoid;
 }
 
-.ui-printContent--questionTable tr {
+.ui-printContent--table tr {
   break-inside: auto;
   break-after: auto;
   page-break-inside: auto;
 }
 
-.ui-printContent--questionTable td,
-.ui-printContent--questionTable th {
-  break-inside: avoid;
-}
-
-.ui-printContent--questionTable {
+.ui-printContent--table {
   display: block;
```

Comment:
Had had changed this in my PR because I noticed it while doing the results UI. If the question text and responses for all questions in a section aren't long enough, the table didn't stretch full width. You can see it in this example here. If you just remove this and leave the table display as `display: table` it should work. Thoughts?

![Screenshot 2024-07-18 at 11 03 12 AM](https://github.com/user-attachments/assets/9ca710e9-7862-4459-85d0-f165ebebae1d)

Reply:
Altho it doesn't seem to be doing it now so maybe it's not needed? Just curious why this was set to block to begin with a guess. I wouldn't think it was needed.

Reply:
Ahh interesting! That would make it... very interesting. And also maybe why I was seeing it still in emulation, but not on the print dialog. Weird!